technic/max_lag.lua

```diff
@@ -1,15 +1,23 @@
-local function explode(sep, input)
-        local t={}
-        local i=0
-        for k in string.gmatch(input,"([^"..sep.."]+)") do
-            t[i]=k
-            i=i+1
-        end
-        return t
-end
+
+local multiplier = tonumber(technic.config:get("max_lag_reduction_multiplier")) or 0.99
```

Comment:
You could also use `technic.config:get_int("max_lag_reduction_multiplier")` here, name should actually be get_number or something like that.
Benefit is that you do not hardcode defaults in code file but in config.lua (and as lesser benefit it validates key during CI tests).